lib/Utils.php

```diff
@@ -317,6 +317,13 @@ public static function parseSchemaAppJsonLd(string $jsonLd, string $baseUri) {
         // @ symbols cause havoc use use associative 
         $jsonObj = \json_decode($jsonLd, true);
 
+        // get context and ensure it is http if it's schema.org, we don't currently support https
+        if (array_key_exists("@context", $jsonObj) && !is_array($jsonObj['@context'])) {
+            $jsonObj['@context'] = str_replace("https://schema.org/", "http://schema.org/", $jsonObj['@context']);
+        } else if (isset($jsonObj['context']['schema'])) {
+            $jsonObj['@context']['schema'] = str_replace("https://schema.org/", "http://schema.org/", $jsonObj['@context']['schema']);
+        }
```

Comment:
Does this need to be done recursively or is it only a problem in the top level element?

Example:  I don't think the nested element in the block below would have the context changed.

```
{
   "@context": "http://schema.org",
   "schema:friend": {
        "@context": "http://schema.org"
    }
}
```

Reply:
Just added a fix for this and a couple test cases that include these contexts.